Mathlib/Algebra/BigOperators/Multiset/Abs.lean

```diff
@@ -0,0 +1,31 @@
+/-
+Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mario Carneiro
```

Comment:
The lemma [was added](https://github.com/leanprover-community/mathlib/commit/a01cf868#diff-ebc9675b853e12be424eeaf5113fda0da4142e290abaf161bc7f2287c2f3df8b) by [Johannes Hölzl](https://github.com/johoelzl) and [Rob Lewis](https://github.com/robertylewis) in 2019, according to [changelog](https://mathlib-changelog.org/v3/theorem/multiset.abs_sum_le_sum_abs).

Reply:
I think we decided the copyright line doesn't change since this is a derivative work. So:
```suggestion
Copyright (c) 2015 Microsoft Corporation. All rights reserved.
Released under Apache 2.0 license as described in the file LICENSE.
Authors: Johannes Hölzl, Rob Lewis
```